### DISCUSSION TITLE
Can I use it to backup my Plex watchstate to a file and restore it when I installed a clean Plex instance?

### DISCUSSION THREAD

**Discussion by Rubeast** [luigi311/JellyPlex-Watched#208] [Q&A]
I'm looking for a way to backup the watchstate of all my users and restore it once Plex is back up and running with a new database.
Is there a way to accomplish this with this application?

**Comment by luigi311** [comment]
There is no way to do this right now, it is technically possible to do it but i havent gotten around to implementing it, issue #130
